zowex/src/main.rs

```diff
@@ -181,18 +200,20 @@ fn run_daemon_command(mut args: String) -> std::io::Result<()> {
         let daemon_proc_info = is_daemon_running();
 
         // when not running, start it.
+        // if daemon_proc_info.is_running == false {
```

Comment:
Can this line be removed?

Reply:
I second Andrew's opinion that the commented line should be removed. It looks like something that was temporarily commented out during testing, and then left by accident afterward.